readthedocs/settings/docker_compose.py

```diff
@@ -239,5 +239,7 @@ def SOCIALACCOUNT_PROVIDERS(self):
     DATA_UPLOAD_MAX_NUMBER_FIELDS = None
     SUPPORT_EMAIL = "support@example.com"
 
+    RTD_FILETREEDIFF_ALL = os.environ("RTD_FILETREEDIFF_ALL").lower() == "true"
```

Comment:
```suggestion
    RTD_FILETREEDIFF_ALL = "RTD_FILETREEDIFF_ALL" in os.environ
```

We are following this pattern in https://github.com/readthedocs/readthedocs.org/blob/aa0b4d1b9ae8f8ba17f35f1cb723b8e36b293070/readthedocs/settings/base.py#L230